modules/ROOT/pages/configure-client-management-azure.adoc

```diff
@@ -63,7 +63,7 @@ When https://docs.microsoft.com/en-us/graph/api/application-post-applications?vi
 
 Client secrets (application password) use a string value in the Azure application to authenticate instead of a certificate for identity. 
 
-Client secrets are initially set to expire six months from the date of creation per Microsoft's security recommendations. You can reset the expiration date for client secrets in Anypoint Platform. The lifetime limit for a client secret is 24 months and can't be extended beyond that. 
+Client secrets are set to expire six months from the date of creation per Microsoft's security recommendations. The lifetime limit for a client secret is 24 months and can't be extended beyond that. 
```

Comment:
Remove as well? Since one can't extend, there's little use explaining 24 months. Except for this I'm good with the PR.

Reply:
Done.